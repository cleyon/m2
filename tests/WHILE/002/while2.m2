@sequence X create
@while @X@ < 6
Iteration @X@
@if @X@ == 4
Exiting while loop early
@break
@endif
Keep going...
@sequence X next
@endwhile
Out of while loop
