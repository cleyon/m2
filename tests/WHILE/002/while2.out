Iteration 0
Keep going...
Iteration 1
Keep going...
Iteration 2
Keep going...
Iteration 3
Keep going...
Iteration 4
Exiting while loop early
Out of while loop
